goose-recipes/triage-issue.yaml

```diff
@@ -0,0 +1,130 @@
+
+  2. Backport a Patch OR Request Clarification
+     This path is for issues that represent a clear bug or CVE that needs a targeted fix.
+
+     2.1. Deep Analysis of the Issue
+     * Use the details extracted from your initial analysis
+     * Focus on keywords and root cause identification
+     * If the Jira issue already provides a direct link to the fix, use that as your primary lead
+
+     2.2. Systematic Source Investigation
+     * Identify the official upstream project and corresponding Fedora package source
+     * Even if the Jira issue provides a direct link to a fix, validate and investigate further as needed
+     * When no direct link is provided, you must proactively search for fixes - do not give up easily
+     * Using the details from your analysis, search these sources in order of priority:
+       - Bug Trackers (for fixed bugs matching the issue description)
+       - Git / Version Control (for commit messages, using keywords, CVE IDs, function names, etc.)
+       - Mailing Lists / Developer Forums (for discussions about the specific problem)
+     * Be thorough in your search - try multiple search terms and approaches based on the issue details
+     * Advanced investigation techniques:
+       - If you can identify specific files, functions, or code sections mentioned in the issue, locate them in the source code
+       - Use git history (git log, git blame) to examine changes to those specific code areas
+       - Look for commits that modify the problematic code, especially those with relevant keywords in commit messages
+       - Check git tags and releases around the time when the issue was likely fixed
+       - Search for commits by date ranges when you know approximately when the issue was resolved
+     * Utilize dates strategically in your search if needed:
+       - Identify the version/release date of the package currently used in RHEL
+       - Focus on fixes that came after the RHEL package version date, as earlier fixes would already be included
+       - For CVEs, use the CVE publication date to narrow down the timeframe for fixes
+       - Check upstream release notes and changelogs after the RHEL package version date
+       - Look for commits between the RHEL package version date and any known fix dates mentioned in comments
+
+     2.3. Validate the Fix
+     * When you think you've found a potential fix, examine the actual content of the patch/commit
+     * Verify that the fix directly addresses the root cause identified in your analysis
+     * Check if the code changes align with the symptoms described in the Jira issue
+     * If the fix doesn't appear to resolve the specific issue, continue searching for other fixes
+     * Don't settle for the first fix you find - ensure it's the right one
+
+     2.4. Validate the fix URL
+     * Make sure to provide a valid URL to the patch/commit.
+     * If the URL is not valid, re-do previous steps.
+
+     2.5. Decide the Outcome
```

Comment:
How does the outcome look from package maintainer perspective? Is it a pull request that may be opened if the agent identifies a fix?